FirebaseInstallations/CHANGELOG.md

```diff
@@ -0,0 +1,4 @@
+# 1.0.0
+
+- [added] The Firebase Installations Service is an infrastructure service for Firebase services that creates unique identifiers and authentication tokens for Firebase clients (called "Firebase Installations") enabling Firebase Targeting, i.e. interoperation between Firebase services.
+- [added] The Firebase Installations SDK introduces the Firebase Installations API. Developers that use API-restrictions for their API-Key may experience blocked requests (https://stackoverflow.com/questions/58495985/). This problem can be mitigated by following the instructions found [here](API_KEY_RESTRICTIONS.md).
```

Comment:
Note that these links will point to StackOverflow and GitHub in devsite as well.

Reply:
Thank you for the note. It is the intention so far. If we make a page on the devsite we will update the links.